lib/rules/media-feature-name-allowed-list/__tests__/index.js

```diff
@@ -39,6 +39,10 @@ testRule({
 		{
 			code: '@media only screen and(min-width: 640px) { }',
 		},
+		{
+			code: '@media ($WIDTH: 50em) { }',
+			description: 'ignore non-standard syntax media feature names',
+		},
```

Comment:
```suggestion
```

This is covered by line 208.

(For future reference, we typically put non-standard cases into their own `testRule` with the appropriate non-standard syntax to separate standard from non-standard.)